README.md

```diff
@@ -6,9 +6,10 @@
 - [Nginx](http://www.nginx.com/)
 - [Envoy](https://www.envoyproxy.io)
 - [Katran (FB)](https://github.com/facebookincubator/katran)
-- [KgLb (Digitalocean)](https://github.com/dropbox/kglb)
+- [KgLb (Dropbox)](https://github.com/dropbox/kglb)
 - [GLB (Github)](https://github.com/github/glb-director)
 - [Traefik](https://github.com/containous/traefik/)
+- [Cilium](https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#maglev-consistent-hashing) - `maglev` support in Kubernetes CNI
```

Comment:
https://cilium.io/use-cases/load-balancer/ ?  or even https://cilium.io ?

Reply:
I wanted to specifically point to the Maglev functionality, as parity with the rest of the doc.

GLB, Unimog, Katran, Google's paper, and then Cilium's maglev code fits with it.

I can duplicate the Cilium reference if you don't mind it twice. I know other people who have missed that Cilium could to the L4LB maglev work because it's buried in the docs.

Reply:
TBH I don't understand why Maglev functionality should be mentioned. It's an implementation detail and all other links are pointing just to the projects not their specific functions.

So yeah, I think keeping just a link to the Cilium makes more sense in this case 😉

Reply:
I found your little repo because I was trying remember something I'd read ages ago, related to the split L4LB using maglev specifically, to have connections survive a L4LB change, as seen in vbernat's doc that I linked.
https://vincent.bernat.ch/en/blog/2018-multi-tier-loadbalancer#second-tier-l4-load-balancing

Specifically, there was a further discussion about ECMP & resilient hashing in the switchgear ahead of the L4LB layer, so that the L4LB layer had happy-path most of the time, and still also handled the ECMP routing decision changing due to quirks.

I did put Cillium in the top post, and put better details into the Posts sections - esp. an extra blogpost during Cillium L4LB development that shows the CPU usage drop from XDP.